### PR TITLE
editorial review of WCF sample

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: samples/wcf/sample.md [hunk 1]
```diff
@@ -2,45 +2,45 @@
...
- * When a message was sent with enter `Response 'Hello from handler'` will be displayed
- * When a message was sent with escape `Request failed due to: 'The request was cancelled after 00:00:05 because no response was received.'` will be displayed
+ * When a message is sent which _does not_ timeout, `Response 'Hello from handler'` is displayed.
+ * When a message is sent which _does_ timeout, `Request failed due to: 'The request was cancelled after 00:00:05 because no response was received.'` is displayed.
 
 
 ## Code walk-through
 
-The sample contains a self-hosted endpoint that leverages the WCF and the Callbacks package to enable reliable WCF services over messaging.
+The sample contains a self-hosted endpoint which uses the [NServiceBus.WCF](/nservicebus/wcf) and [NServiceBus.Callbacks](/nservicebus/messaging/callbacks.md) packages to achieve reliable WCF services over messaging.
```
Comment (bording) [inline samples/wcf/sample.md]:
```suggestion
The sample contains a self-hosted endpoint which uses the [NServiceBus.Wcf](/nservicebus/wcf) and [NServiceBus.Callbacks](/nservicebus/messaging/callbacks.md) packages to achieve reliable WCF services over messaging.
```

File: samples/wcf/sample.md [hunk 2]
```diff
@@ -2,45 +2,45 @@
...
 
-The [WCF](/nservicebus/wcf)) and the [Callback](/nservicebus/messaging/callbacks.md) component are enabled. The WCF component is configured to cancel requests after five seconds and a named pipe binding is used to expose the service as shown below:
+The WCF package is configured to cancel requests after five seconds and a named pipe binding is used to expose the service:
```
Comment (bording) [inline samples/wcf/sample.md]:
```suggestion
The WCF package is configured to cancel requests after five seconds, and a named pipe binding is used to expose the service:
```

File: samples/wcf/sample.md [hunk 3]
```diff
@@ -41,6 +41,6 @@ The request message is passed to the proxy:
 
 snippet: wcf-proxy-call
 
-The proxy will asynchronously wait until the response or the cancellation is received. A response is sent from a regular handler hosted in the same endpoint:
+The proxy asynchronously waits until the response, or the cancellation is received. A response is sent from a regular handler hosted in the same endpoint:
```
Comment (adamralph) [inline samples/wcf/sample.md]: @danielmarbach on a second look I think this should be:

> The proxy waits, asynchronously, until the response or cancellation is received.

Comment (danielmarbach) [reply samples/wcf/sample.md]: @adamralph maybe fix it directly on master?

**Pull request merged**